src/debugger/nodeDebugWrapper.ts

```diff
@@ -43,6 +43,7 @@ interface ILaunchArgs {
     platform: string;
     target?: string;
     internalDebuggerPort?: any;

```

Comment:
I did it this way because the actual value/type is user provided. Initially I had said that it was a number, but then I couldn't `parseInt` even though it works fine at runtime to ensure that we end up with a number.